src/businessrules/loaders/CartLoader.java

```diff
@@ -1,11 +1,70 @@
 package businessrules.loaders;
 
+import businessrules.dai.AddonRepository;
+import businessrules.dai.CustomerRepository;
+import businessrules.dai.FoodRepository;
 import entities.Cart;
+import entities.Customer;
+import entities.Food;
+import entities.Selection;
+import org.json.JSONArray;
+import org.json.JSONException;
 import org.json.JSONObject;
 
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
 public class CartLoader {
 
-    public static Cart loadCart(JSONObject rawData){
-        return null;
+    CustomerRepository customerRepository;
+    CustomerLoader customerLoader;
+    FoodLoader foodLoader;
+    SelectionLoader selectionLoader;
+    FoodRepository foodRepository;
```

Comment:
I suggest keeping repositories together in attributes list for sake of consistency and organization